### DISCUSSION TITLE
Rule Chaining

### DISCUSSION THREAD

**Discussion by gurukriven** [CacheControl/json-rules-engine#329] [General]
How to achieve rule chaining? Suppose a fact is fed into the rule and the rule updates the fact, this updated fact is actually going to trigger another rule (based on the updated fact) , does the engine takes care of that rule execution as well ?

**Comment by chris-pardy** [comment]
Updating a fact won't trigger rules to re-run. You should set a priority, but if you need a rule to run twice in an execution you'll need to do that manually.
